oak_proto_rust/build.rs

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2023 The Project Oak Authors
+// Copyright 2024 The Project Oak Authors
```

Comment:
(no need to update copyright -- actually technically it should be the earliest date, not the latest)

Reply:
This file was created in 2024? Should 2023 be the copyright date for the whole `oak_proto_rust` crate>